docker/Dockerfile-github

```diff
@@ -0,0 +1,9 @@
+FROM jetscape/base:v1.3
```

Comment:
It would be interesting to see if this workflow can be completed first for use in the other workflows that also compile jetscape.

Reply:
Here, can we first trigger a build of the dockerfile.build image, and then use that image in the FROM statement?

Reply:
Currently the Jetscape base build is a manual process, I am not sure what I had in mind for that comment.  Once the base build has been completed and uploaded to Docker then another manual trigger would be needed to use the new base image here in GitHub.  How stable is the base image for our purposes?

Reply:
The base image is probably somewhat stable -- but periodically will likely need updating. Is there not a way we can wait for the base build to complete before building the deploy build on top of it? Can we not just put another push action in the yaml file and it will execute sequentially?

Reply:
Yes, automating the base image build would be ideal. However on second thought let's focus on getting this deploy workflow finished first. Thanks